exasol_advanced_analytics_framework/udf_communication/local_discovery.py

```diff
@@ -0,0 +1,56 @@
+import socket
+import time
+from typing import cast
+
+from exasol_advanced_analytics_framework.udf_communication.local_discovery_socket import LocalDiscoverySocket
+from exasol_advanced_analytics_framework.udf_communication.messages import PingMessage
+from exasol_advanced_analytics_framework.udf_communication.peer_communicator import PeerCommunicator
+from exasol_advanced_analytics_framework.udf_communication.serialization import serialize_message, deserialize_message
+
+
+class LocalDiscovery:
+
+    def __init__(self,
+                 discovery_timeout_in_seconds: int,
+                 peer_communicator: PeerCommunicator,
+                 local_discovery_socket: LocalDiscoverySocket
+                 ):
+        self._local_discovery_socket = local_discovery_socket
+        self._peer_communicator = peer_communicator
+        self._discovery_timeout_in_minutes = discovery_timeout_in_seconds
```

Comment:
look wrong. shouldn't be:
```suggestion
        self._discovery_timeout_in_minutes = discovery_timeout_in_seconds / 60
```
?

Reply:
should be fixed with rework of the LocalDiscovery